### DISCUSSION TITLE
Installing with non-standard cmake installation location

### DISCUSSION THREAD

**Discussion by choidami** [google/or-tools#2537] [Build and configuration issues] [answered]
Hello,

I am trying to install or-tools on a cluster, where the cmake installed in /usr/bin is an older version than the one required by or-tools. I installed a new version of cmake in a non-standard location, but I'm not sure how to get or-tools to use this cmake instead of the one in /usr/bin. I can't just get rid of /usr/bin in PATH. I'm not very familiar with make files-- is there something I can change in the makefiles to specify a specific cmake path?

**Comment by choidami** [comment, marked as answer]
I was being stupid. I can just prepend PATH with the new path.

**Reply by Mizux** [reply]
Glad you find a solution ! no one is stupid ;)
